src/map/mob.cpp

```diff
@@ -2082,6 +2097,13 @@ static int mob_ai_sub_lazy(struct mob_data *md, va_list args)
 	md->last_thinktime=tick;
 
 	if (md->master_id) {
+		// Get mob data of master
```

Comment:
I know but I needed this change to be able to debug map_search_freecell to test if it's working, because otherwise it would have been called constantly.